### ISSUE TITLE
Empty tables disappears

### ISSUE THREAD

**Issue by ppecio** [loopindex/ckeditor-track-changes#130]
When i add new table in content which is not makred as a change (is a black regular text), click somewhere out of the editor and then click back into editor (but not on change) table disappears. After some investigation I found that selection cleaning up function is causing this, which calls _cleanupAroundNode where dom.hasNoTextOrStubContent returns true for insertion/deletion with empty table inside.

**Comment by imdfl** [comment]
Yes, the cleanup is too aggressive. Not sure I have the resource to address this now, but I'll keep it in mind (or wait for the community to step in...)
